HISTORY.rst

```diff
@@ -8,6 +8,7 @@ Unreleased Changes
   pygeoprocessing source distribution.  This fixes an issue where files
   matching a variety of extensions anywhere in the pygeoprocessing directory
   might be included with the source distribution.
+* Adding GDAL 3 support and dropping GDAL 2 support.
```

Comment:
Can you indicate that the only non-backwards compatible issues in GDAL 2 to GDAL 3 were the introduction of `osr.CreateCoordinateTransformation` (vs `CoordinateTransformation`) and the need to set `[srs_ref].osr.SetAxisMappingStrategy(osr.OAMS_TRADITIONAL_GIS_ORDER)` because of https://trac.osgeo.org/gdal/wiki/rfc73_proj6_wkt2_srsbarn#Axisorderissues? I think this will only be an issue for people who manually construct

Reply:
I definitely will, but wanted to note that `osr.CoordinateTransformation` still works in GDAL 3, but since we're not supporting GDAL 2, can move on to the newer recommended function.